official/transformer/transformer_estimator_benchmark.py

```diff
@@ -100,10 +101,12 @@ def _report_benchmark(self,
       exp_per_sec = sum(exp_per_second_list) / (len(exp_per_second_list))
       metrics.append({'name': 'exp_per_second',
                       'value': exp_per_sec})
-    self.report_benchmark(
-        iters=eval_results['global_step'],
-        wall_time=wall_time_sec,
-        metrics=metrics)
+
+    flags_str = flags_core.get_nondefault_flags_as_str()
```

Comment:
Thanks for adding FLAGs to the estimator version!

Now the only thing left (that I really want) is the Resnet keras version.